src/Entity/User.php

```diff
@@ -134,7 +134,7 @@ public function getUsername(): string
         return $this->username;
     }
 
-    public function setUsername(string $username): void
+    public function setUsername(?string $username): void
```

Comment:
Likewise, should we call `setUsername` with null?

Reply:
same as displayName